pkgs/by-name/st/stem/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchurl
+, python3
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "stem";
+  version = "1.8.2";
+
+  src = fetchurl {
+    url = "https://gitlab.torproject.org/tpo/network-health/stem/-/archive/master/stem-master.zip";
+    sha256 = "sha256-qHvg/T8oU9vtyPWK/fXxIPJlg1FPCc76SzdOTw5iNcQ=";
+  };
+
+  nativeBuildInputs = with python3.pkgs; [ setuptools ];
+
+  dontUseSetuptoolsCheck = true;
```

Comment:
whats the reason ? no tests ? or failing tests ?

Reply:
Failing tests.

Reply:
`doCheck = false` with a comment, is more common way of disabling tests

Reply:
changes made.